lib/proj-parser.js

```diff
@@ -31,15 +31,15 @@ function determineDotnetVersions(rootDir) {
         '/TargetFrameworks defined in the Project.PropertyGroup field of ' +
         'your .csproj file');
     }
-    frameworks = _.compact(_.map(versions, toReadableVersion));
+    frameworks = _.compact(_.map(versions, toReadableFramework));
   });
   if (frameworks.length < 1) {
     throw new Error('Could not find valid/supported .NET version');
```

Comment:
While we are here, how about making this message more accurate with `'Could not find valid/supported .NET Target Framework in .csproj file '` 

Does this mean we completely don't support any other *proj files here?

Reply:
For the first comment - I will be addressing that in what I am working on now - https://snyksec.atlassian.net/browse/BST-319 - reading TF from other files. As it currently stands, the message is accurate that we couldnt find a TF.

Yes, we completely don't support any other *proj files here.